src/TypeSpec.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -989,3 +1032,29 @@ export function getFormattedType(program: Program, type: Type): FormattedType {
         encode: encodeData
     } as FormattedType;
 }
+
+// This is a temporary solution. After we uptake getAllModels we should delete this.
+export function navigateModels(
+    context: SdkContext,
+    namespace: Namespace,
+    models: Map<string, InputModelType>,
+    enums: Map<string, InputEnumType>
+) {
+    const computeModel = (x: Type) =>
+        getInputType(
+            context,
+            getFormattedType(context.program, x),
+            models,
+            enums
+        ) as any;
+    const skipSubNamespaces = isGlobalNamespace(context.program, namespace);
+    navigateTypesInNamespace(
+        namespace,
+        {
+            model: (x) =>
+                x.name !== "" && !["AuthFlow", "MyFlow"].includes(x.name) && x.kind === "Model" && computeModel(x),
```

Comment:
Here `!["AuthFlow", "MyFlow"].includes(x.name)` is a temporary workaround. Will try to move it before merge.